### DISCUSSION TITLE
会不会有人编译一个PCLmium出来，不包括附加内容就是纯净的PCL给人用？

### DISCUSSION THREAD

**Discussion by MFn233** [Meloong-Git/PCL#3949] [提问] [answered]
如题，这个问题有点像知乎里的人才会提出来

**Comment by Pigeon0v0** [comment]
这是否属于违反 LICENSE 第二条...

**Reply by MFn233** [reply]
哦我的朋友，但是我就是很好奇
毕竟vscode有vscodium
chrome有chromium
各种东西都有各种东西ium

**Reply by wuliaodexiaoluo** [reply]
> 哦我的朋友，但是我就是很好奇 毕竟vscode有vscodium chrome有chromium 各种东西都有各种东西ium

PCLium 是吧
GitHub Reply 终于不会放到下面了

**Comment by LTCatt** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/blob/879fc00d72be74038018d684415c73db54c78292/LICENCE#L20
